### ISSUE TITLE
Since the new release, our exceptions are not being respected

### ISSUE THREAD

**Issue by wesleytodd** [nodesecurity/nsp#207]
We run nsp in sub-directories of our project, but keep one `.nsprc` at the root with exceptions.  It previously respected it, but, it seems, not since the release ~2 hours ago.

**Comment by fwielstra** [comment]
I'm running into the same issue, just installed NSP for the first time; do you have comments in your .nsprc file? Removing those fixed it for me. May be it no longer uses `rc` like the documentation implies.

**Comment by willdurand** [comment]
Indeed, `rc` is not used anymore 😢

**Comment by fwielstra** [comment]
Maybe there was a security issue in `rc`? I think we're onto something 😲

**Comment by nicholascapo** [comment]
There are no comments in our `.nsprc`
Copying it to the current directory right before running `nsp check` does avoid this issue.

Does the replacement for `rc` support walking up the file path looking for an `.nsprc`?
In other words if the files are organized like this:
```
.nsprc
serviceA/package.json
serviceA/[...].js
serviceB/package.json
serviceB/[...].js
```

Does running `nsp check` in directory `serviceA` find and use the `.nsprc.` in the parent directory?

**Comment by nlf** [comment]
you're right! i overlooked this, i'll get that support added back in. i don't know if i'll bother using `rc` again, but we can walk up looking for a config file for sure

**Comment by willdurand** [comment]
Comments are rather useful to know why exceptions are there.

**Comment by wolfgang42** [comment]
:+1: for adding comments. At the very least, print an error message if the `.nsprc` fails to parse so we can tell that something went wrong!

**Comment by kumar303** [comment]
Also, it would have been helpful if `nsp check` failed with an error telling me that it couldn't parse `.nsprc` due to comments. Running with `--verbose` doesn't seem to add any extra information.

**Comment by phy51c5** [comment]
Adding to the others in requesting the restoration of the ability to add comments to a .nsprc exception file.

**Comment by nicholascapo** [comment]
Any word on this?
